### PR TITLE
Feat: Claim Seed Neurons

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Function to claim seed neurons. Not exposed yet to the user.

# Changes

* Add service "seed-neurons" with single function.
* Bump nns-js. Needed to use the Genesis Token Canister.

# Tests

* This is temporary until users all the proper users claim their neurons.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker @mstrasinskis please review

This is the function I mentioned earlier. Let's add this to the debug flow in another PR once @mstrasinskis adds the new changes.

**Review by mstrasinskis — APPROVED** [review]
LGTM. A couple of minor notes.

File: frontend/svelte/src/lib/services/seed-neurons.services.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import type { Secp256k1PublicKey } from "../keys/secp256k1";
+import { getLedgerIdentityProxy } from "../proxy/ledger.services.proxy";
+import { accountsStore } from "../stores/accounts.store";
+import { createAgent } from "../utils/agent.utils";
+import { mapNeuronErrorToToastMessage } from "../utils/error.utils";
+import { translate } from "../utils/i18n.utils";
+
+const buf2hex = (buffer: ArrayBuffer): string => {
+  return [...new Uint8Array(buffer)]
+    .map((x) => x.toString(16).padStart(2, "0"))
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/svelte/src/lib/services/seed-neurons.services.ts:13]: Yes, good point. But the types don't match. This starts as an ArrayBuffer, the other as a `number[]`, but I should be able to use bytesToHexString if first I convert it with `[...new Uint8Array(buffer)]`.

Let me check

File: frontend/svelte/src/lib/services/seed-neurons.services.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    const identity = await getLedgerIdentityProxy(hardwareWallet?.identifier);
+    const governance = await GenesisTokenCanister.create({
+      agent: await createAgent({ identity, host: HOST }),
+    });
+
+    const bufferKey = identity.getPublicKey() as Secp256k1PublicKey;
+    const hexPubKey = buf2hex(bufferKey.toRaw());
+    const isHex = hexPubKey.match("^[0-9a-fA-F]+$");
+    if (!isHex) {
+      throw new Error(`${hexPubKey} is not a hex string.`);
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/svelte/src/lib/services/seed-neurons.services.ts]: done

File: frontend/svelte/src/lib/services/seed-neurons.services.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/svelte/src/lib/services/seed-neurons.services.ts]: done

File: frontend/svelte/src/lib/services/seed-neurons.services.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/svelte/src/lib/services/seed-neurons.services.ts]: Thank you.

**Pull request merged**